chunky/src/java/se/llbit/chunky/ui/render/tabs/LightingTab.java

```diff
@@ -129,6 +149,21 @@ public LightingTab() throws IOException {
 
     sunColor.colorProperty().addListener(sunColorListener);
 
+    modifySunTexture.setTooltip(new Tooltip("Change whether the the color of the sun texture is modified by the apparent sun color"));
+    modifySunTexture.selectedProperty().addListener((observable, oldValue, newValue) -> {
+      scene.sun().setEnableTextureModification(newValue);
+      apparentSunColor.setDisable(!newValue);
+    });
+
+    apparentSunColor.setDisable(true);
+    apparentSunColor.colorProperty().addListener(apparentSunColorListener);
+
+    sunRadius.setName("Sun radius");
+    sunRadius.setTooltip("Radius of the sun");
```

Comment:
Maybe "size" to explain radius? Do we need this tool tip?

Reply:
I changed the name to `Sun size`, which is self explanatory. I'll remove the tooltip.